wine-classification/{{cookiecutter.project_name}}/workflows/example.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import flytekit.extras.sklearn
+
+from sklearn.datasets import load_wine
+from sklearn.linear_model import LogisticRegression
+from flytekit import task, workflow
+
+@task
+def get_data() -> pd.DataFrame:
+    """Get the wine dataset."""
+    return load_wine(as_frame=True).frame
+
+@task
+def process_data(data: pd.DataFrame) -> pd.DataFrame:
+    """Simplify the task from a 3-class to a binary classification problem."""
+    return data.assign(target=lambda x: x["target"].where(x["target"] == 0, 1))
+
+@task
+def train_model(data: pd.DataFrame, hyperparameters: dict) -> LogisticRegression:
+    """Train a model on the wine dataset."""
+    features = data.drop("target", axis="columns")
+    target = data["target"]
+    return LogisticRegression(max_iter=3000, **hyperparameters).fit(features, target)
+
+@workflow
+def training_workflow(hyperparameters: dict) -> LogisticRegression:
```

Comment:
we should always have a set of default parameters for our template workflows - otherwise it'll be hard to test.

Reply:
OK